### DISCUSSION TITLE
Full width resizable table example link is broken

### DISCUSSION THREAD

**Discussion by Ratandeep13** [TanStack/table#4243] [General]
https://tanstack.com/table/v8/docs/examples/react/full-width-resizable-table
<img width="1361" alt="Screenshot 2022-07-31 at 9 01 52 PM" src="https://user-images.githubusercontent.com/16688493/182033783-4c08f172-6783-40d7-bb40-d1fb80db5223.png">

**Comment by StarHosea** [comment]
here load sanbox failed

**Comment by talatkuyuk** [comment]
The example is working now, but the resizing is a bit weird. Actually, in full width tables, when you resize a column, only the current and adjacent column widths have to change and the others must keep their widths, am I right?
